SimMuon/MCTruth/python/NewMuonAssociatorByHits_cfi.py

```diff
@@ -0,0 +1,147 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
ok, the wonderful 'new's can be tolerated for a few weeks.  We'll see after pre1 validation what would be gained with other releases.